src/lib/operators/aggregate_sort.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include <boost/container/pmr/polymorphic_allocator.hpp>
+#include <boost/container/scoped_allocator.hpp>
+#include <boost/functional/hash.hpp>
+#include <functional>
+#include <limits>
+#include <memory>
+#include <optional>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "abstract_aggregate_operator.hpp"
+#include "abstract_read_only_operator.hpp"
+#include "expression/aggregate_expression.hpp"
+#include "resolve_type.hpp"
+#include "storage/abstract_segment_visitor.hpp"
+#include "storage/reference_segment.hpp"
+#include "storage/value_segment.hpp"
+#include "types.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+/*
+Operator to aggregate columns sort-based by certain functions, such as min, max, sum, average, and count. The output is a table
```

Comment:
This comment doesn't tell me a lot on how the aggregate operates. The fact that it calls `Sort` internally is something that should be documented.